crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -1,8 +1,11 @@
 use std::ops::Mul;
 
 use super::Transform;
+
+#[cfg(feature = "bevy-support")]
 use bevy_ecs::{component::Component, reflect::ReflectComponent};
 use bevy_math::{Affine3A, Dir3, Mat4, Quat, Vec3, Vec3A};
+#[cfg(feature = "bevy-support")]
 use bevy_reflect::{std_traits::ReflectDefault, Reflect};
```

Comment:
I am a little concerned about this increasing the maintenance burden. It's going to be really easy to add an import in the wrong place here, and break the `!bevy-support` version.

How can we tests for this in CI?

Reply:
Let's make a follow-up issue for adding CI testing for all of our standalone crates: bevy_reflect, bevy_math, bevy_color and assuming this PR is merged, bevy_transform.

Reply:
https://github.com/bevyengine/bevy/issues/13611 I took the liberty